cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -990,7 +1031,8 @@ def _add_head_node(self):
                     "enable_intel_hpc_platform": "true" if self.config.is_intel_hpc_platform_enabled else "false",
                     "cw_logging_enabled": "true" if self.config.is_cw_logging_enabled else "false",
                     "cluster_s3_bucket": self.bucket.name,
-                    "cluster_config_s3_key": f"{self.bucket.artifact_directory}/configs/cluster-config.yaml",
+                    "cluster_config_s3_key": f"{self.bucket.artifact_directory}/configs/"
```

Comment:
minor: let's avoid mixing concatenation and f-strings. Just use format in cases like this.